src/providers/ipa/ipa_init.c

```diff
@@ -231,6 +231,17 @@ static errno_t ipa_init_dyndns(struct be_ctx *be_ctx,
     return EOK;
 }
 
+static bool ipa_check_fqdn(const char *str)
+{
+    const char ch = '.';
+    char *ret;
+    ret = strchr(str, ch);
+    if (ret != NULL) {
+        return true;
+    }
+    return false;
```

Comment:
Is there a reason to have all this complexity instead of just something like:`return strchr(str, '.') ? true : false;`

Reply:
or just `return strchr(str, '.')`, I'm fine with having this in a function so that it's easier to see what the code is doing from the caller, but all those declarations seem just excessive sorry.

Reply:
@jhrozek Good with `return strchr(str, '.') ;`

Reply:
I'm fine with that, feel free to use that version and push new patches..